src/main/resources/application.properties

```diff
@@ -1 +1,11 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=1234
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.jpa.hibernate.ddl-auto=create
 
+spring.h2.console.enabled=true
+
+logging.level.org.hibernate.SQL=debug
+logging.level.org.hibernate.type.descriptor.sql.BasicBinder=trace
```

Comment:
please don't forget to remove code that helped you in your local development. Such code should not be in production code